src/game_actors.cpp

```diff
@@ -32,26 +32,24 @@ Game_Actors::Game_Actors() {
 Game_Actors::~Game_Actors() {
 }
 
-void Game_Actors::Fixup() {
+void Game_Actors::SetSaveData(std::vector<lcf::rpg::SaveActor> save) {
```

Comment:
> Remove Main_Data::game_data.actors 

Intential copy?

Reply:
Yes, internally we move the `SaveActor` objects.

The new model for loading save data will be that you move it.